release-schema.json

```diff
@@ -0,0 +1,102 @@
+{
+  "definitions": {
+    "Amendment": {
+      "properties": {
+        "unstructuredChanges": {
+          "title": "Unstructured changes",
+          "description": "Unstructured or semi-structured changes to an earlier notice.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/UnstructuredChange"
+          },
+          "uniqueItems": true,
+          "wholeListMerge": true
+        }
+      }
+    },
+    "UnstructuredChange": {
+      "title": "Unstructured change",
+      "description": "Unstructured or semi-structured change to an earlier notice.",
+      "type": "object",
+      "properties": {
+        "relatedLot": {
+          "title": "Related lot",
+          "description": "The identifier of the lot to which this change relates, if any.",
+          "type": [
+            "string",
+            "null"
+          ]
+        },
+        "oldValue": {
+          "title": "Old value",
+          "description": "The value of the data element referenced by the where field, before the change.",
```

Comment:
Can you add backticks (\`) around where? (Same for others)

Reply:
Done, so is the definition of `UnstructuredChangeValue`.